iree/turbine/kernel/wave/analysis/index_sequence_analysis.py

```diff
@@ -749,11 +757,16 @@ def get_index(custom: CustomOp):
         lhs = get_custom(custom.lhs)
         rhs = get_custom(custom.rhs)
 
-        lhs_index = get_index(lhs)
-        rhs_index = get_index(rhs)
+        lhs_index = get_index(lhs) if lhs.indexing_dims else None
```

Comment:
How about modifying this inside the `get_index` function itself to reduce redundant code